screengrab/build.gradle

```diff
@@ -1,24 +1,25 @@
-// Top-level build file where you can add configuration options common to all sub-projects/modules.
+apply plugin: 'com.github.ben-manes.versions'
```

Comment:
adds `./gradlew dependencyUpdates` where you can have a look at newer versions

Reply:
Is this worth the external dependency?

Reply:
Yeah it's only a buildscript dependency. The built aar won't have any effects.